front/scripts/update_assistants_models.ts

```diff
@@ -20,31 +17,23 @@ async function updateWorkspaceAssistants(
     where: { workspaceId },
```

Comment:
We can probably refine this query to add the where clause on the `modelId` now 🙃.

Reply:
😆 indeed